--- libmemcached/dump.cc.orig	2020-12-11 14:48:24.323929773 +0200
+++ libmemcached/dump.cc	2020-12-11 14:53:01.743945101 +0200
@@ -47,7 +47,7 @@
 static memcached_return_t ascii_dump(Memcached *memc, memcached_dump_fn *callback, void *context, uint32_t number_of_callbacks)
 {
   /* MAX_NUMBER_OF_SLAB_CLASSES is defined to 200 in Memcached 1.4.10 */
-  for (uint32_t x= 0; x < 200; x++)
+  for (uint32_t x= 0; x < 64; x++)
   {
     char buffer[MEMCACHED_DEFAULT_COMMAND_SIZE];
     int buffer_length= snprintf(buffer, sizeof(buffer), "%u", x);
