--- certbot-apache/certbot_apache/override_gentoo.py	2018-12-06 02:31:07.000000000 +0200
+++ certbot-apache/certbot_apache/override_gentoo.py.new	2018-12-23 11:27:09.605976748 +0200
@@ -19,7 +19,7 @@
         vhost_files="*.conf",
         logs_root="/var/log/apache2",
         ctl="apache2ctl",
-        version_cmd=['apache2ctl', '-v'],
+        version_cmd=['apache2', '-v'],
         restart_cmd=['apache2ctl', 'graceful'],
         restart_cmd_alt=['apache2ctl', 'restart'],
         conftest_cmd=['apache2ctl', 'configtest'],
