--- a/init.d/src/gentoo-backuppc	2017-03-12 20:48:56.924734956 +0200
+++ b/init.d/src/gentoo-backuppc	2017-03-12 20:50:02.232732811 +0200
@@ -1,4 +1,4 @@
-#!/sbin/runscript
+#!/sbin/openrc-run
 #
 # DESCRIPTION
 #
@@ -21,7 +21,7 @@
 start() {
 	checkconfig || return 1
 	ebegin "Starting BackupPC"
-	start-stop-daemon --start --chuid ${USER} --user ${USER} --pidfile ${PID_FILE} --exec ${EXEC} -- ${EXEC_OPTIONS}
+	start-stop-daemon --start --user ${USER} --pidfile ${PID_FILE} --exec ${EXEC} -- ${EXEC_OPTIONS}
 	eend $?
 }
 
